src/lib.rs

```diff
@@ -640,6 +641,93 @@ impl Picture {
     pub fn offset(&self) -> i64 {
         self.inner.pic.m.offset
     }
+
+    /// Chromaticity coordinates of the source colour primaries.
+    pub fn color_primaries(&self) -> pixel::ColorPrimaries {
+        unsafe {
+            #[allow(non_upper_case_globals)]
+            match (*self.inner.pic.seq_hdr).pri {
+                DAV1D_COLOR_PRI_BT709 => pixel::ColorPrimaries::BT709,
+                DAV1D_COLOR_PRI_UNKNOWN => pixel::ColorPrimaries::Unspecified,
+                DAV1D_COLOR_PRI_BT470M => pixel::ColorPrimaries::BT470M,
+                DAV1D_COLOR_PRI_BT470BG => pixel::ColorPrimaries::BT470BG,
+                DAV1D_COLOR_PRI_BT601 => pixel::ColorPrimaries::BT470BG,
+                DAV1D_COLOR_PRI_SMPTE240 => pixel::ColorPrimaries::ST240M,
+                DAV1D_COLOR_PRI_FILM => pixel::ColorPrimaries::Film,
+                DAV1D_COLOR_PRI_BT2020 => pixel::ColorPrimaries::BT2020,
+                DAV1D_COLOR_PRI_XYZ => pixel::ColorPrimaries::ST428,
+                DAV1D_COLOR_PRI_SMPTE431 => pixel::ColorPrimaries::P3DCI,
+                DAV1D_COLOR_PRI_SMPTE432 => pixel::ColorPrimaries::P3Display,
+                DAV1D_COLOR_PRI_EBU3213 => pixel::ColorPrimaries::Tech3213,
+                DAV1D_COLOR_PRI_RESERVED => unreachable!(),
```

Comment:
Is that indeed unreachable? Also below in the same places

Reply:
With current dav1d git master yes, I've checked these appear only in the header file. Maybe we could make these methods return an `Option<>`?

Reply:
@lu-zero What do you think?

Reply:
The hacky way is to return `pixel::ColorPrimaries::Unspecified` as well, since the user would have no mean to deal with it anyway and add a mean to get the raw bitstream value in case the user knows how to use it since there are few "for future use" values. that assuming PRI_RESERVED maps to the bitstream value.

Anybody willing to read `ISO/IEC 23091-4/ITU-T H.273` and doublecheck?

Reply:
This one? https://www.itu.int/rec/T-REC-H.273-202309-P/en

I'm not registered in TIES.

Reply:
https://www.itu.int/rec/T-REC-H.273-201612-S/en is freely accessible

Reply:
Well, in the spec the reserved value is specified as a range, for instance, for ColorPrimaries, it is `23-255`, while `DAV1D_COLOR_PRI_RESERVED` is 255. Same goes for TransferCharacteristics (`19-255`) and MatrixCoefficients (`15-255`).

Reply:
I'd say that mapping it to `Unspecified` wouldn't feel too bad. Maybe I could future-proof av-data and add a Reserved(u8) but that's unrelated to this patch.